CONFLICTS.md

```diff
@@ -35,3 +35,10 @@ references related issues.
    `Sylius\Bundle\ApiBundle\Swagger\AcceptLanguageHeaderDocumentationNormalizer` 
    references this class throws an exception
   `Class "ApiPlatform\Core\Metadata\Resource\ResourceNameCollection" not found`
+
+- `friends-of-behat/symfony-extension:^2.4`:
+
+   Some scenarios with anonymous customers started to fail after this release. Potentially it is related to
+   [this PR](https://github.com/FriendsOfBehat/SymfonyExtension/pull/190), but it need to be investigated (and fixed)
+
+   Example failing build: https://github.com/Zales0123/Sylius/actions/runs/3590609125/jobs/6044176431 
```

Comment:
Same on Sylius 1.11 so I've opened a PR to the `1.11` branch with this conflict: https://github.com/Sylius/Sylius/pull/14610